packages/react/src/Dialog.stories.tsx

```diff
@@ -7,7 +7,7 @@ import {default as Dialog} from './Dialog'
 /* Dialog Version 1*/
 
 export default {
-  title: 'Components/DialogV1',
+  title: 'Deprecated/Components/DialogV1',
```

Comment:
If we're comfortable doing this for Dialog, I think it would be good to do the same for [Tooltip](https://primer.style/react/storybook/?path=/story/components-tooltip--default). This was mentioned during today's Primer Patterns working session.

Reply:
Yup! I had drafted a PR for this and just flipped it to review
https://github.com/primer/react/pull/4760